.github/workflows/basic.yml

```diff
@@ -64,6 +64,15 @@ jobs:
             shared_library: 'ON'
             level_zero_provider: 'ON'
             install_tbb: 'ON' 
+          # - os: 'ubuntu-22.04'
```

Comment:
hah 😄 wrong merge conflict resolution, I guess - you're just adding extra comments here

Reply:
exactly :) done